ttkwidgets/font/familylistbox.py

```diff
@@ -55,11 +55,12 @@ def _on_click(self, *args):
     def selection(self):
         """
                 Selection property.
-        
+
         :return: None if no font is selected and font family name if one is selected.
         :rtype: None or str
         """
         selection = self.listbox.curselection()
+        print(selection)
```

Comment:
Could you remove the print statement here?

Reply:
Thanks for noticing, I often forget to delete my debug print statement.